libs/search-widget/src/core/utils.ts

```diff
@@ -208,6 +208,27 @@ export function mapSmartParagraph2WidgetParagraph(paragraph: Search.FindParagrap
   } as WidgetParagraph;
 }
 
+export function mapParagraph2SmartParagraph(paragraph: Search.Paragraph): Search.FindParagraph {
```

Comment:
is this function coming from somewhere else? if so we should delete the code from the other place to avoid duplicated code.

Reply:
no, I did it myself, I did not copy, I swear